src/pages/Legal/TermsOfService.tsx

```diff
@@ -0,0 +1,400 @@
+              fees.
+            </RawText>
+          </RawText>
+          <RawText as='p'>
+            <RawText as='strong'>Sending and Receiving Cryptocurrency:</RawText> You may also send
+            and receive cryptocurrency on the Platform. To do so, click on the &quot;Assets&quot;
+            tab, and then the applicable asset that you wish to send or receive. Once on an asset's
+            page, you'll see buttons for both send and receive functions. When inputting addresses
+            to send or receive, we recommend only copying and pasting addresses (or utilizing the QR
+            code) to minimize the possibility of errors or typos. We also highly encourage you to
+            triple check any addresses before sending or receiving. Again, because the blockchain is
+            immutable and we do not control it, we will not be liable for any errors you cause when
+            sending and receiving.
+          </RawText>
+          <RawText as='dl' display={'flex'} flexDirection={'column'} rowGap={4}>
+            <RawText as='dt'>
+              <RawText as='strong'>Interactions with other smart contracts:</RawText> In addition to
+              the above functions on the Platform, you may also interact with other types of
+              third-party smart contracts for additional functionality of your cryptocurrency. Many
+              of these additional functions are commonly referred to as decentralized finance or
+              &quot;
+              <RawText as='strong'>DeFi</RawText>&quot; and include functions such as earning yield
+              on your cryptocurrency. As with trading on the Platform, these functions are entirely
+              maintained by third-parties that are not affiliated with us, so again, please do your
+              own due diligence before interacting with any DeFi functions on our Platform.
+            </RawText>
+            <RawText as='dd' marginInlineStart={{ base: 2, lg: 4 }}>
+              <RawText as='strong'>Quick note on &quot;APR&quot;:</RawText> When you see &quot;
+              <RawText as='strong'>APR</RawText>&quot; on the Platform, it means the estimated
+              annual percentage for a particular smart contract function. The APR for a given smart
+              contract function on our Platform is pulled from that smart contract's publicly
+              available data. Accordingly, you acknowledge that we are not responsible for your
+              reliance on any APR figures that you see on our Platform. For more information on how
+              the underlying APR is calculated, please refer to the website or documentation of the
+              third-party who maintains the applicable smart contract.
+            </RawText>
+          </RawText>
+          <RawText as='p'>
+            <RawText as='strong'>Alpha Platform:</RawText> We have made available an alternate
```

Comment:
IMO, this entire section regarding `alpha.shapeshift.com` should be removed until it's hosted, and references to the mobile apps should be removed until we host an open source version of the apps, as the current versions are not hosted via the DAO, and we should not blur any lines there.